src/Literate.jl

```diff
@@ -260,6 +260,8 @@ function create_configuration(inputfile; user_config, user_kwargs, type=nothing)
                        !get(user_config, "execute", cfg["execute"]) ?
                        ("````@example $(get(user_config, "name", replace(cfg["name"], r"\s" => "_")))" => "````") :
                        ("````julia" => "````")
+    cfg["mime_extensions"] = [(MIME("image/png"), ".png"), (MIME("image/jpeg"), ".jpeg"),
```

Comment:
Perhaps we don't even have to make this configurable? If you want to keep it, perhaps call it `image_formats` or something?

Reply:
Perhaps leave it undocumented for now and if someone have problems with exporting svg by default we can add this option later.

Reply:
I think it would be better to keep this, since new (image) formats appear all the time and handling them is transparent to Literate (+ Documenter, Franklin, etc). Eg JPEGXL, HEIC, etc would work just fine if they can be saved by the application, and can be captured with this option. 

Also, this option can handle more than images, eg a table saved in a spreadsheet format. So I would keep it as is.

Reply:
But if you insist I am happy to remove and/or rename this, just say so.

Reply:
Fair enough, but what happens with format other than image formats when inserted using the `![...](...)` format? I thought that was only for images in markdown?

Reply:
For example, `text/html` and `text/markdown` is handled in a different way

Reply:
Good point, renamed it.